pkg/storage/chunk/cache/background.go

```diff
@@ -118,7 +119,10 @@ func (c *backgroundCache) writeBackLoop() {
 				return
 			}
 			c.queueLength.Sub(float64(len(bgWrite.keys)))
-			c.Cache.Store(context.Background(), bgWrite.keys, bgWrite.bufs)
+			err := c.Cache.Store(context.Background(), bgWrite.keys, bgWrite.bufs)
+			if err != nil {
+				continue
```

Comment:
Could you log the error?

Reply:
ok, thanks .
https://github.com/grafana/loki/pull/5083

Wait for this PR to be completed and then I would make changes. These two PRs are to modify the cache module, and there are many file conflicts.